inc/interface-wpseo-wordpress-ajax-integration.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO
+ */
+
+if ( ! interface_exists( 'WPSEO_WordPress_AJAX_Integration' ) ) {
```

Comment:
I think this is not necessary. We don't check for existence in our code base.

Reply:
Ah I see. We have wrapped that one (`WPSEO_WordPress_Integration`) because we did introduce that in Premium first and after a while that interface has been copied to free.